theforeman.org/pipelines/lib/packaging.groovy

```diff
@@ -34,6 +34,49 @@ def find_changed_packages(diff_range) {
     return changed_packages.split()
 }
 
+def find_changed_debs(diff_range) {
+    def changed_debs = []
+
+    def changed_dependencies = find_added_or_changed_files(diff_range, 'dependencies/*/*/changelog').split()
+    def changed_plugins = find_added_or_changed_files(diff_range, 'plugins/*/changelog').split()
+    def changed_core = find_added_or_changed_files(diff_range, 'debian/*/*/changelog').split()
+
+    for(core in changed_core) {
+        (type, os, project) = core.split('/')
```

Comment:
in https://github.com/theforeman/jenkins-jobs/pull/96#discussion_r701655615, I used `(folder, os, project)`, as it felt like using `type` here will welcome someone to refactor the line 3 down (`type: 'core'`) to read `type: type` and that'd break things.

not blocking on this, but I think not calling this thing "type" is cleaner.

Reply:
type type type